packages/@aws-cdk/aws-codedeploy/lib/config.ts

```diff
@@ -0,0 +1,47 @@
+export enum ComputePlatform {
```

Comment:
Add jsdocs to all public APIs. The easiest (and most effective way) is to copy & paste from AWS docs.